gnovm/pkg/gnolang/nodes.go

```diff
@@ -1707,6 +1805,30 @@ func (sb *StaticBlock) GetIsConst(store Store, n Name) bool {
 	}
 }
 
+func (sb *StaticBlock) getAt(store Store, path ValuePath) *StaticBlock {
```

Comment:
duplicate logic with another func: `GetBlockNodeForPath`.

Reply:
This was screwed up during the merge. Good catch.